src/BaseApiAbstract.php

```diff
@@ -22,6 +22,7 @@
  */
 abstract class BaseApiAbstract
 {
+    const TTL_PREDICTION_CORRECTION_TIME_SEC = 10;
```

Comment:
I don't see where do you use it. Word `prediction` may confuse it this context. Maybe just `TTL_CORRECTION_TIME_SEC`. And const must be in `AuthTokenProvider` class, then its name will be clear.